### PR TITLE
Correct CI

### PR DESCRIPTION
**Pull request by clem21**

### CODE REVIEW

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -44,12 +44,15 @@ jobs:
...
-      - run: mkdir -p /home/runner/.composer/cache
+      - run: mkdir -p /home/runner/{.composer/cache,.config/composer}
         if: steps.cache-composer.outputs.cache-hit != 'true'
 
       - name: Composer v2
         run: sudo composer self-update --2
 
+      - name: Validate composer.json
+        run: composer validate --strict
+
```
Comment (jacquesbh) [inline .github/workflows/tests.yaml:55]: This is done below.

```
run: make test.composer
```

**Review by jacquesbh — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yaml [hunk 2]
```diff
@@ -44,12 +44,12 @@ jobs:
...
           restore-keys: composer2-php:${{ matrix.php }}-
 
-      - run: mkdir -p /home/runner/.composer/cache
+      - run: mkdir -p /home/runner/{.composer/cache,.config/composer}
         if: steps.cache-composer.outputs.cache-hit != 'true'
 
       - name: Composer v2
         run: sudo composer self-update --2
-
+
```
Comment (jacquesbh) [inline .github/workflows/tests.yaml:52]: Oops?

**Review by jacquesbh — CHANGES_REQUESTED** [review]

**Review by maximehuran — APPROVED** [review]

**Review by lanfisis — APPROVED** [review]

**Review by jacquesbh — CHANGES_REQUESTED** [review]
We could use a little rebase here :).

**Review by jacquesbh — APPROVED** [review]
Thank you @clem21! 🎉

**Pull request merged**